qa/tasks/cephfs/fuse_mount.py

```diff
@@ -305,6 +305,9 @@ def umount_wait(self, force=False, require_clean=False, timeout=900):
         """
         :param force: Complete cleanly even if the MDS is offline
         """
+        if not (self.is_mounted() and self.fuse_daemon):
```

Comment:
Jos, let's add a debug message that the mount is not unmounted. That makes it easier to confirm the PR fixes the problem and leaves a notice to future debuggers.

Reply:
added a message